### PR TITLE
fix(FormControl): updated select styling in light and dark themes

### PR DESCRIPTION
**Pull request by thatblindgeye**
Towards https://github.com/patternfly/patternfly/issues/5695 and #5751 

Need to test this update in other OS/browser combos to ensure nothing visually breaks.

Would most likely need a v6 update as well if we go forward with this.

### CODE REVIEW

File: src/patternfly/components/FormControl/form-control.scss [hunk 1]
```diff
@@ -273,13 +273,18 @@
       --#{$form-control}--PaddingRight: calc(var(--#{$form-control}__select--PaddingRight) - 1px);
       --#{$form-control}--PaddingLeft: calc(var(--#{$form-control}__select--PaddingLeft) - 4px);
     }
+
+    // We need this property for certain Linux distros/non-Mac OS where dark theme styles aren'y applied properly.
```
Comment (thatblindgeye) [inline src/patternfly/components/FormControl/form-control.scss:277]:
```suggestion
    // We need this property for certain Linux distros/non-Mac OS where dark theme styles aren't applied properly.
```

**Comment by mcoker** [comment]
For reference, we dug up an old issue that reported this same bug a couple of years ago and was fixed, and that fix was later removed.

https://github.com/patternfly/patternfly/issues/4772
https://github.com/patternfly/patternfly/pull/4773/files

Looks like it probably [broke in this PR](https://github.com/patternfly/patternfly/pull/5531) where we changed  `color: var(--#{$form-control}--placeholder--child--Color);` to `--#{$form-control}--Color: var(--#{$form-control}--m-placeholder--child--Color);`

**Review by mcoker — APPROVED** [review]
Tested Chrome, FF, Safari, and Edge on my mac - the backgrounds and item colors all look good (unchanged)

**Comment by thatblindgeye** [comment]
Checked Windows + Chrome/FireFox/Edge and from what I can tell nothing breaks either. Can confirm that the bug from #5751 does affect Windows + Chrome as well.

**Review by srambach — APPROVED** [review]
LGTM thank you!

**Pull request merged**